### PR TITLE
KO: translation for site/ko/tutorials/distribute/save_and_load.ipynb

### PR DESCRIPTION
**Pull request by Kimminjae748**
Korean translation for distribute/save_and_load.ipynb
Thanks for reviewing!

### CODE REVIEW

**Comment by Kimminjae748** [comment]
I signed it.

-----Original Message-----

Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

📝 Please visit https://cla.developers.google.com/ to sign.

Once you've signed (or fixed any issues), please reply here with @googlebot I signed it! and we'll verify it.

What to do if you already signed the CLA
Individual signers
It's possible we don't have your GitHub username or you're using a different email address on your commit. Check your existing CLA data and verify that your email is set on your git commits.
Corporate signers
Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to go/cla#troubleshoot (Public version).
The email used to register you as an authorized contributor must be the email used for the Git commit. Check your existing CLA data and verify that your email is set on your git commits.
The email used to register you as an authorized contributor must also be attached to your GitHub account.
ℹ️ Googlers: Go here for more info.

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 1]
```diff
@@ -0,0 +1,576 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "A0lG6qgThxAS"
+   },
+   "source": [
+    "## 개요\n",
+    "\n",
+    "훈련 도중 모델을 저장하고 로드하는 것은 흔히 일어나는 일입니다. 케라스 모델을 저장하고 로딩하기 위한 API에는 2가지가 있는데: high-level API와 low-level API가 해당합니다. 이 튜토리얼은 `tf.distribute.Strategy`를 사용할 때 어떻게 SavedModel APIs를 사용할 수 있는지 보여줍니다. SavedModel와 직렬화에 관한 일반적인 내용을 학습하려면, [saved model guide](../../guide/saved_model.ipynb)와 [Keras model serialization guide](../../guide/keras/save_and_serialize.ipynb)를 읽어보는 것을 권장합니다. 간단한 예로 시작해보겠습니다:"
```
Comment (paulkim1997) [inline site/ko/tutorials/distribute/save_and_load.ipynb]: '해당합니다' -> '해당됩니다'

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 2]
```diff
@@ -0,0 +1,576 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "L01wjgvRizHS"
+   },
+   "source": [
+    "## 모델 저장하고 로딩하기\n",
+    "이제 사용할 모델을 가지고 있으므로 API 저장/로딩에 대해 살펴봅시다. \n",
+    "두 가지 API를 사용 가능합니다:\n",
```
Comment (paulkim1997) [inline site/ko/tutorials/distribute/save_and_load.ipynb]: '가능합니다' -> '할 수 있습니다'

**Comment by lamberta** [comment]
Looks like there are still some review comments to resolve---is that correct?

**Comment by rickiepark** [comment]
Hi @Kimminjae748 ,
Please check the review comments for your PR.

**Comment by lamberta** [comment]
Thank you.

Community translations are now located in the [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n) repo (see [this message](https://groups.google.com/a/tensorflow.org/g/docs/c/GJPq7S_fw4c/m/lA1qfBX2EQAJ) for context).

I'll move over/duplicate existing pull requests as I find time. In the meantime, please feel free to create a new pull request in that repo 😀 Then you can close this PR and add a link to the new one.

**Comment by lamberta** [comment]
MIgrated in https://github.com/tensorflow/docs-l10n/pull/83

**Pull request closed without merging**